### DISCUSSION TITLE
Word curation

### DISCUSSION THREAD

**Discussion by danconde** [azadbolour/eagerwords-community#5] [Q&A] [answered]
I do not want to question the choice of specific set of words in a certain level, but I mention this in case there may be some problem with the algorithm.  A beginner bot used the words "ENE" and "DOIN"  The first can be "east-northeast", but the latter is only listed as a contraction for "doing" and I do not feel the latter are appropriate for a beginner bot to use.  Perhaps some filtering went incorrectly?

**Comment by azadbolour** [comment, marked as answer]
- According to the server logs, the word ENE was played by the master bot. And indeed ENE appears towards the end of the curated ordered list of words.

- The word DOIN does indeed appear early in the word ordering. The reason for that is that the word ordering is generally based on the frequency of the word stem (technically the word's "lemma"). So DO, DOES, DONE, DOIN, and DOTH all appear together in the ordering. 

You are right that it is odd to see DOIN or worse DOTH played by the beginner bot. But I think it makes sense to include DOES and DONE. EagerWords relies on an externally curated word list that includes word frequencies and the lemma function. Fine-tuning the word order is currently beyond the scope of EagerWords' capabilities.

**Comment by danconde** [comment]
I understand with regards to fine tuning word order.  I do not recall setting the bot level to the master, as I have used the Basic level as the highest level. There's a chance I accidentally set the level to master, but I find it unlikely.
